src/overview/components/ResultList.jsx

```diff
@@ -27,6 +27,15 @@ const timeGapToSpaceGap = makeNonlinearTransform({
 })
 
 const ResultList = ({searchResult}) => {
+
+    //if no result found display relavant message
+    if(searchResult.rows.length === 0){
+        return <p className={styles.message}>
```

Comment:
I'm very free about style. I don't care about semicolon, spaces, and many other things. This is one of the few things I had a real issue with but the final choice for me is your.